concept/type/owns.feature

```diff
@@ -0,0 +1,3952 @@
+    Then <root-type>(<subtype-name>) get owns overridden(username) get label: name
+    Then <root-type>(<subtype-name>) get owns, with annotations (DEPRECATED): <annotation>; contain:
+      | username |
+    Then <root-type>(<subtype-name>) get owns, with annotations (DEPRECATED): <annotation>; do not contain:
+      | name |
+    Then <root-type>(<subtype-name>) get owns contain:
+      | username |
+    Then <root-type>(<subtype-name>) get owns do not contain:
+      | name |
+    Examples:
+      | root-type | supertype-name | subtype-name | annotation       | value-type |
+      | entity    | person         | customer     | key              | string     |
+      | entity    | person         | customer     | unique           | string     |
+      | entity    | person         | customer     | subkey(LABEL)    | string     |
+      | entity    | person         | customer     | values("1", "2") | string     |
+      | entity    | person         | customer     | range("1", "2")  | string     |
+      | entity    | person         | customer     | card(1, 2)       | string     |
+      | entity    | person         | customer     | regex("\S+")     | string     |
+      | relation  | description    | registration | key              | string     |
+      | relation  | description    | registration | unique           | string     |
+      | relation  | description    | registration | subkey(LABEL)    | string     |
+      | relation  | description    | registration | values("1", "2") | string     |
+      | relation  | description    | registration | range("1", "2")  | string     |
+      | relation  | description    | registration | card(1, 2)       | string     |
+      | relation  | description    | registration | regex("\S+")     | string     |
+
+    # TODO: Maybe it should be rejected?
+  Scenario Outline: <root-type> types can re-override owns with <annotation>s
+    When put attribute type: email
+    When attribute(email) set value-type: <value-type>
+    When attribute(email) set annotation: @abstract
+    When put attribute type: work-email
+    When attribute(work-email) set value-type: <value-type>
+    When attribute(work-email) set supertype: email
+    When <root-type>(<supertype-name>) set annotation: @abstract
+    When <root-type>(<supertype-name>) set owns: email
+    When <root-type>(<supertype-name>) get owns(email) set annotation: @<annotation>
+    When <root-type>(<subtype-name>) set annotation: @abstract
+    Then <root-type>(<subtype-name>) get owns contain: email
+    Then <root-type>(<subtype-name>) get owns(email) get annotations contain: @<annotation>
+    When <root-type>(<subtype-name>) set owns: work-email
+    When <root-type>(<subtype-name>) get owns(work-email) set override: email
+    # TODO: These commas, semicolons, and colons are a mess and are different for different subcases. Need to refactor it!
+    Then <root-type>(<subtype-name>) get owns overridden(work-email) get label: email
```

Comment:
Not sure about `get owns overridden`, the question is connected to `override` itself.